test/classes/delete-free/lifetimes/borrow-escapes.good

```diff
@@ -2,7 +2,9 @@ borrow-escapes.chpl:45: In function 'bad1':
 borrow-escapes.chpl:48: error: Scoped variable global would outlive the value it is set to
 borrow-escapes.chpl:46: note: consider scope of r
 borrow-escapes.chpl:52: In function 'bad2':
-borrow-escapes.chpl:57: error: Scoped variable outer would outlive the value it is set to
+borrow-escapes.chpl:57: error: Scoped variable would outlive the value it is set to
```

Comment:
I would keep the name of the variable in the first error line. I prefer clarity over brevity in this case.

Reply:
The split initialization changes caused this difference in error message but I didn't think it worth investigating at the current time. I have other work adjusting the lifetime checker. If you like we can make an issue to revisit it.

Reply:
(this can happen when the lifetime checker sees an offending variable that is a temp. It leaves the names of the temps off.)